### PR TITLE
Normalized the package number to PEP440

### PR DESCRIPTION
**Pull request by hmaarrfk**
Now set to `0.15.dev0` instead of `0.15dev` since you used to keep
getting the warning:

```
[...]/site-packages/setuptools/dist.py:398: UserWarning: Normalizing
'0.15dev' to '0.15.dev0'
```

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/__init__.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
 pkg_dir = osp.abspath(osp.dirname(__file__))
 data_dir = osp.join(pkg_dir, 'data')
 
-__version__ = '0.15dev'
+__version__ = '0.15.dev0'
```
Comment (soupault) [inline skimage/__init__.py]: Not sure is the trailing zero is useful. Maybe change to `0.15.dev`?

**Comment by hmaarrfk** [comment]
The trailing 0 is the canonical version number.

I can add a some notes in the release docs.

File: RELEASE.txt [hunk 2]
```diff
@@ -42,7 +42,10 @@ make a PR to update these notes after you are done with the release. ;-)
   - Once the consensus is found, ask the project maintainers to merge the PR.
 
 - In the master branch, update the version number in ``skimage/__init__.py``
-  to the next ``-dev`` version, commit, and push.
+  to the next ``.dev0`` version, commit, and push. This should follow PEP440
+  meaning that the appropriate version number would look something like
+  ``0.20.dev0`` with the period beteween ``20`` and ``dev`` and a trailing
+  ``0`` immediately after ``dev``.
```
Comment (soupault) [inline RELEASE.txt]: Awesome!

**Comment by soupault** [comment]
I hope this merge will not break the website... :)
Thanks @hmaarrfk !

**Pull request merged**